pip-dep/requirements.txt

```diff
@@ -7,7 +7,8 @@ phe~=1.4.0
 Pillow~=6.2.2
 requests~=2.22.0
 scipy~=1.4.1
-syft-proto~=0.2.9.a1
+#syft-proto~=0.2.9.a1
+git+git://github.com/OpenMined/syft-proto@vvm/placeholder-shape#egg=syft_proto
```

Comment:
Is there another PR to go with this? Don't see one on `syft-proto`.

Reply:
Oops. Created: https://github.com/OpenMined/syft-proto/pull/87